src/ripple/peerfinder/tests/PeerFinder_test.cpp

```diff
@@ -0,0 +1,152 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <BeastConfig.h>
+#include <ripple/peerfinder/impl/Logic.h>
+#include <beast/unit_test/suite.h>
+#include <beast/chrono/manual_clock.h>
+
+namespace ripple {
+namespace PeerFinder {
+
+class Logic_test : public beast::unit_test::suite
+{
+public:
+    struct TestStore : Store
+    {
+        std::size_t
+        load (load_callback const& cb) override
+        {
+            return 0;
+        }
+
+        void
+        save (std::vector <Entry> const&) override
+        {
+        }
+    };
+
+    struct TestChecker
```

Comment:
A comment here would be helpful: /*Mock for class that tests remote listening sockets to make sure they are connectable (See PeerFinder::Checker for a concrete example) */

I don't think you need one on "TestStore" above because it inherits from Store, so I can see what it's mocking.